scripts/install_third_party_libs.py

```diff
@@ -36,7 +36,10 @@
     ('future', '0.18.2', os.path.join('third_party', 'python_libs')),
     ('six', '1.16.0', os.path.join('third_party', 'python_libs')),
     ('certifi', '2021.10.8', os.path.join(TOOLS_DIR, 'certifi-2021.10.8')),
-    ('typing-extensions', '4.0.1', os.path.join('third_party', 'python_libs'))
+    # This library is used very frequently so we install it to default location.
+    # If installed in third_party, it does not work correctly, unless we add
+    # it to path in many different places.
+    ('typing-extensions', '4.0.1', '')
```

Comment:
Can you please test whether this works if you install it into `os.path.join('third_party', 'python_libs')`

Reply:
@vojtechjelinek If it is installed to `os.path.join('third_party', 'python_libs')`, all the imports fail unless the path is added to each of the files where typing_extensions is imported using the `sys.path.insert` call.

Reply:
Hmm, that is weird, can you post the exact imports that are failing?

Reply:
@vojtechjelinek Can you check the logs here on line 3268.
https://github.com/oppia/oppia/runs/6455287883?check_suite_focus=true

Reply:
I think moving the
```
# Add third_party to path. Some scripts access the scripts layer even before
# python_libs is added to path.
_THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party', 'python_libs')
sys.path.insert(0, _THIRD_PARTY_PATH)
```
code from the third party to common.py (as ti was before) and see if it fixes the problem.

Reply:
@vojtechjelinek The typing_extension import is used in _feconf.py_ which is imported in some files (for example ./setup.py) without importing _common.py_. Hence, I had to shift the `_THIRD_PARTY_PATH` to _feconf.py_.